pallets/crowdloan/src/migrations/migrate_add_contributors_count.rs

```diff
@@ -0,0 +1,90 @@
+use alloc::string::String;
+use frame_support::{BoundedVec, traits::Get, weights::Weight};
+
+use crate::*;
+
+pub fn migrate_add_contributors_count<T: Config>() -> Weight {
```

Comment:
Could we get away without the migration since the crowdloan code didn't get to mainnet?

Reply:
It's likely dependent on the next release date.